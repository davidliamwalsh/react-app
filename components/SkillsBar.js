import { Component } from 'react'

class SkillsBar extends Component {

  render () {
    return <div className="c-skills">
      <h2 className="c-main__head">My Skills</h2> 
      <hr className="c-main__head-line" />

      <div className="c-skills__container">
        <div className="c-skills__container__wrapper">
          <div className="c-skills__container__wrapper--html">
            HTML
            <span className="c-skills__container__wrapper--right">95%</span>
          </div>
        </div>
          
        <div className="c-skills__container__wrapper">
          <div className="c-skills__container__wrapper--css">
            CSS
            <span className="c-skills__container__wrapper--right">80%</span>
          </div>
        </div>

        <div className="c-skills__container__wrapper">
          <div className="c-skills__container__wrapper--javascript">
          <span>JScript</span>
            <span className="c-skills__container__wrapper--right">64%</span>
          </div>
        </div>

        <div className="c-skills__container__wrapper">
          <div className="c-skills__container__wrapper--react">
            React
            <span className="c-skills__container__wrapper--right">25%</span>
          </div>
        </div>

        <div className="c-skills__container__wrapper">
          <div className="c-skills__container__wrapper--ror">
            RoR
            <span className="c-skills__container__wrapper--right">50%</span>
          </div>
        </div>
      </div>
    </div>
  }
}

export default SkillsBar