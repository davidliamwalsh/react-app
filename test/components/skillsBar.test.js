import SkillsBar from '../../components/SkillsBar'

describe('<SkillsBar />', () => {
  it('it renders correctly', () => {
    
  })
}) 