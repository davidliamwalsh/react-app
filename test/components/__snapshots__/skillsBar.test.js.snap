// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SkillsBar /> it renders correctly 1`] = `ShallowWrapper {}`;